upup/pkg/fi/cloudup/dotasks/droplet.go

```diff
@@ -219,3 +221,40 @@ func (_ *Droplet) CheckChanges(a, e, changes *Droplet) error {
 	}
 	return nil
 }
+
+type terraformDropletOptions struct {
+	Image    *string                  `cty:"image"`
+	Size     *string                  `cty:"size"`
+	Region   *string                  `cty:"region"`
+	Name     *string                  `cty:"name"`
+	Tags     []string                 `cty:"tags"`
+	SSHKey   []string                 `cty:"ssh_keys"`
+	UserData *terraformWriter.Literal `cty:"user_data"`
+	VPCUUID  *string                  `cty:"vpc_uuid"`
```

Comment:
This field is never being set in RenderTerraform

Reply:
sure, updated. Most of my testing, that was set to empty and was not being used. I have a note to test the vpc and Load balancer feature separately and more PR's to follow.